vsts/pipelines/templates/_agentCleanUpJobTemplate.yml

```diff
@@ -2,12 +2,19 @@ parameters:
   agentName: ''
 
 jobs:
+
 - job: Cleaning_${{ parameters.agentName }}
   displayName: ${{ parameters.agentName }}
   pool:
     name: OryxLinux
     demands:
     - agent.name -equals ${{ parameters.agentName }}
   timeoutInMinutes: 30
+
   steps:
+
+  - script: echo Cleaning `pwd`; find . -type d -name coverage.cobertura.xml -print -exec chmod +w {} \;
```

Comment:
what is this xml file?? how does it get generated?imo if we really need to clean it .. we should clean it where it gets generated ..